dao/src/main/kotlin/repositories/analyzerrun/DaoAnalyzerRunRepository.kt

```diff
@@ -76,8 +79,23 @@ class DaoAnalyzerRunRepository(private val db: Database) : AnalyzerRunRepository
 
             createAnalyzerConfiguration(analyzerRun, config)
 
-            projects.forEach { createProject(analyzerRun, it) }
-            packages.forEach { createPackage(analyzerRun, it) }
+            val projectsMap = projects.associateBy({ it.identifier }) { createProject(analyzerRun, it) }
```

Comment:
Nit: Personally, I find the following syntax a bit more readable

    associate { it.identifier to createProject(analyzerRun, it) }